ios/Classes/AblyFlutterStreamHandler.h

```diff
@@ -0,0 +1,21 @@
+@import Foundation;
+@import Flutter;
+#import "AblyFlutterPlugin.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AblyFlutterStreamHandler : NSObject<FlutterStreamHandler>
+
++(instancetype)new NS_UNAVAILABLE;
+-(instancetype)init NS_UNAVAILABLE;
+
+-(instancetype)initWithAbly:(AblyFlutterPlugin *const)plugin NS_DESIGNATED_INITIALIZER;
+
+@property(nonatomic, readonly) AblyFlutterPlugin * plugin;
+
+- (FlutterError* _Nullable)onListenWithArguments:(id _Nullable)arguments
```

Comment:
Really minor thing but in Objective-C there are scenarios, and I think this is one of them, where you can prefix the type with `nullable` which is slightly nicer to read than `_Nullable` ... i.e. `-(nullable FlutterError *)onListenWithArguments:(nullable id)arguments`. Slightly prettier, is all.

Also, cleaner to call the `events` argument name placeholder `eventSink` to match the selector signature. This also makes it a bit clearer that it's a block.

Reply:
Handled in 1542ced08593476576c72f0a4262282e34967b10